Firestore/Source/Local/FSTLocalSerializer.mm

```diff
@@ -128,7 +130,8 @@ - (FSTPBWriteBatch *)encodedMutationBatch:(FSTMutationBatch *)batch {
 
   FSTPBWriteBatch *proto = [FSTPBWriteBatch message];
   proto.batchId = batch.batchID;
-  proto.localWriteTime = [remoteSerializer encodedTimestamp:batch.localWriteTime];
+  proto.localWriteTime = [remoteSerializer
+      encodedTimestamp:Timestamp{batch.localWriteTime.seconds, batch.localWriteTime.nanoseconds}];
```

Comment:
Objective-C does not have overloads that differ only in type of argument so you'll need to name these methods differently.

Meanwhile when `batch.localWriteTime` becomes a `Timestamp` we'll need to update this because the property accessors will need a `()` on the end, at which point we can just perform the simplification.

Reply:
Ah, I missed the `()`. sgtm.